py/embedded-server/deephaven_server/start_jvm.py

```diff
@@ -59,24 +72,24 @@ def start_jvm(
     # Append user-created args, allowing them to override these values
     system_properties.update(jvm_properties)
 
-    # Build the classpath and expand wildcards, if any
-    jvm_classpath = [os.path.join(os.path.dirname(__file__), 'jars', '*.jar')]
-    jvm_classpath.extend(classpath)
-    jvm_classpath = _expandWildcardsInList(jvm_classpath)
+    # Expand the classpath, so a user can resolve wildcards
+    expanded_classpath = list(itertools.chain.from_iterable(iglob(e, recursive=True) for e in extra_classpath))
```

Comment:
@jmao-denver - can you give this a lookover - I think it replaces the previous functionality. I've tested it out, and it seems to work.